data/building-hours/9-osa.yaml

```diff
@@ -0,0 +1,17 @@
+name: OSA
+subtitle: Office of Student Activities
```

Comment:
Actually, I withdraw my complaint. 

Are there any cases left that use the abbreviation field?